### DISCUSSION TITLE
[V5] node ace:make model command not working.

### DISCUSSION THREAD

**Discussion by mdjamilaj** [adonisjs/core#2642] [Help] [answered]
I have try to create an model create using command line in adonisjs v5.
i run the command **node ace:make model User** show the error.
Cannot find module 'C:\Users\abir2\Desktop\crm-backend\ace:make

**Comment by RomainLanz** [comment, marked as answer]
Hey @mdjamilaj! 👋 

You forgot a space in your command, it must be `node ace make:model User`, not `node ace:make model User`.

**Reply by mdjamilaj** [reply]
![image](https://user-images.githubusercontent.com/57256323/118770970-c3421500-b8a3-11eb-9adc-58b52890b4b7.png)
https://adonisjs.com/adonisjs-at-a-glance
I am following this page

**Reply by RomainLanz** [reply]
Well that is a typo. Fixed now.

**Comment by biswa34** [comment]
![Screenshot_1](https://user-images.githubusercontent.com/58202181/120926131-43b4a280-c6f9-11eb-9c11-31e319310997.png)

Got this issue of making Model using Ace in Adonis v5

**Reply by RomainLanz** [reply]
Ensure to have installed and setup the `lucid` package.

**Reply by biswa34** [reply]
The previous version of Adonis i.e V4 is better than this. Setup is taking a lot time in New version.

**Reply by RomainLanz** [reply]
It takes ~4 seconds.
It is not because you do not read the documentation that it is worse.

**Reply by biswa34** [reply]
Can you help me on one thing. 
I have two different table for admins and users and want to have separated Auth.

In short I want to create an application with Adonis that Have 2 Login. One for User and another for SuperAdmin and other for Admin.

How can I configure that ?
I tried  by adding admin guard on auth but got this error.
![Screenshot_2](https://user-images.githubusercontent.com/58202181/120926747-d8b89b00-c6fb-11eb-85e1-60a471cec479.png)
![Screenshot_3](https://user-images.githubusercontent.com/58202181/120926748-d9e9c800-c6fb-11eb-8f57-42b013017b8e.png)

**Reply by biswa34** [reply]
Can you help me with my problem ? It will be very much helpful.

**Reply by mdjamilaj** [reply]
Install Lucid & try
[Follow the docs](https://docs.adonisjs.com/guides/database/introduction)

**Reply by managerkoffi** [reply]
in V5 it seems adonis not configuring the ORM at start. Here is a video to help deploy it. [https://www.youtube.com/watch?v=gyEWAf8wYuk](https://www.youtube.com/watch?v=gyEWAf8wYuk )

**Comment by mahdigh99** [comment]
in mac installation didn't complete well and User and Token Model and Migrations doesn't create! 
and make:model , make:migration command doesn't work!

<img width="1087" alt="Screen Shot 1401-02-20 at 16 08 56" src="https://user-images.githubusercontent.com/24169864/167620416-f39fa615-a819-42ff-914d-5aff7a13086f.png">

**Reply by RomainLanz** [reply]
You need to install the database module.

https://docs.adonisjs.com/guides/database/introduction

**Comment by mahdigh99** [comment]
No, I mean the starter package is not complete!
however when i install in windows it's work correctly but in mac when I use :
`npm init adonis-ts-app@latest hello-world`
It's not Implement User Model and Migration and i can't use Make:Model Command.
however I install  database module and it's not working again!
<img width="1100" alt="Screen Shot 1401-02-20 at 16 24 35" src="https://user-images.githubusercontent.com/24169864/167622617-f70aad47-3c01-4ced-943d-b611b1688987.png">

**Reply by mahdigh99** [reply]
i do node ace configure @adonisjs/lucid 
and it's work but my problem not solved because i need User  and Token Default model and migrations.

**Reply by mahdigh99** [reply]
Thanks,
I find it.
It changed from Adonis V.4
you have to use : npm i @adonisjs/auth & node ace configure @adonisjs/auth

**Reply by magic-thomas** [reply]
adonisjs need more specific guide for v5. 

the new version looks much far away from v4 and laravel way.

**Comment by EdrissAria** [comment]
> I have try to create an model create using command line in adonisjs v5. i run the command **node ace:make model User** show the error. Cannot find module 'C:\Users\abir2\Desktop\crm-backend\ace:make

According to adonisjs documentation: 
Assuming you already have Lucid [set up](https://docs.adonisjs.com/guides/database/introduction) , run the following command to create your first data model.
`node ace make:model User`

first set up locid then run above command
run `npm i @adonisjs/lucid` 
then configure lucid by running `node ace configure @adonisjs/lucid`
it works like a charm

**Comment by arislaode** [comment]
I face the same problem. Before you run `node ace make:model User`  you have to make sure to install `npm i @adonisjs/lucid` and running  configure database for adonisjs `node ace configure @adonisjs/lucid`. After that, you ready to create new model  `node ace make:model Users` . Check node ace --help to make sure command `make:model` is exist.
